### ISSUE TITLE
social auth redirects to login page after login (Google Oauth2)

### ISSUE THREAD

**Issue by vovkd** [python-social-auth/social-app-django#75]
For some reason social started redirect to login page even after authorization (user created, access code and refresh_token retrived). It worked perfectly and then just started redirect all new users back to login page, so they can't login

**Comment by iamjagjeetubhi** [comment]
Please help me, I have the same error.
Everything was working before but now user redirects to login page again.
here is my website https://thepoet.me
I have created another app on facebook and it is working good on localhost.

**Comment by deinokrates** [comment]
Did anyone find a solution to this issue?

**Comment by vovkd** [comment]
For me problem was in another part of the system

**Comment by tiffanyqi** [comment]
Same issue. Any updates?

**Comment by tiffanyqi** [comment]
I've managed to pinpoint it to a Key Error for '_auth_user_id' in the get_user method in my virtualenv's django/contrib/auth/__init__.py. Anyone know how to bypass the KeyError / not get an auth_user_id error? Thanks!

**Comment by tiffanyqi** [comment]
Fixed by changing is_active default to True rather than False--thinking Django itself updated something

**Comment by mattwilsoncp** [comment]
I just went into /admin and deleted the user and then it started working again.

**Comment by sachitad** [comment]
Hey guys, I recommend using django-allauth. Here is a quick tutorial to help you guys: https://www.youtube.com/watch?v=ZTBexYIIOP8

**Comment by usudaysingh** [comment]
Hi Guys, social-auth-login was working fine.But suddenly it does not. It redirects user to login page after all authentication and creation process user creation, retrieve tokens and save extra data on server. While it is working fine on local.And onserver gmail gives warning that website is insecure while on local it does not.

Can anyone please help...

**Comment by usudaysingh** [comment]
> Fixed by changing is_active in my User model to default to True rather than False--thinking Django itself updated something

Hi tiffanyqi, i also tried this but not working.And other suggestion ?

**Comment by chriscauley** [comment]
TLDR; I had a typo in my AUTHENTICATION_BACKENDS. 

I went through the login flow for google and it looked like nothing happened (worked great with github). I then loaded the session in the command line and it looked like the user was logged in. I eventually realized the session's authentication back end didn't match any of the entries in settings.AUTHENTICATION_BACKENDS, which means the session thought the user was logged in, but django was ignoring it. Fixing the typo fixed it.

**Comment by vicentesuc** [comment]
i solved this , doing the next.

First check your login 
{% url 'social:begin' 'google-oauth2' %}

remove if you have  ?next={{ request.path }}

dont use at the same time
  
SOCIAL_AUTH_LOGIN_REDIRECT_URL=''
LOGIN_REDIRECT_URL

just one of these..

add a path('/{your_url}')

i hope, this help to you
